scenarios/triso/deployment_calcs/test_existing_lwr.py

```diff
@@ -0,0 +1,99 @@
+import os
+import pandas as pd
+import pytest
+import tempfile
+import xml.etree.ElementTree as ET
```

Comment:
Okay, here I see you imported it as ET.  Is the other script missing it?

Reply:
It is!